src/js/LottoNumbers.js

```diff
@@ -0,0 +1,30 @@
+const { RESULT_MESSAGE } = require('./constants.js')
+
+const LottoNumbers = {
+    // 1부터 45까지의 숫자 중 6개를 랜덤으로 선택하는 함수
+    generateRandomNumbers() {
+        const numbers = Array.from({ length: 45 }, (_, i) => i + 1);
+        const randomNumbers = [];
+
+        while (randomNumbers.length < 6) {
+            const randomIndex = Math.floor(Math.random() * numbers.length);
+            const randomNumber = numbers.splice(randomIndex, 1)[0];
+            randomNumbers.push(randomNumber);
+        }
+
+        return randomNumbers.sort((a, b) => a - b);
+    },
+
+    getNumbers(numLottos) {
+        let list = [];
+        for (let i = 0; i < numLottos; i++) {
+            const lottoNumbers = this.generateRandomNumbers(numLottos);
+            console.log(lottoNumbers)
+            list.push(lottoNumbers)
+        }
```

Comment:
1. 객체의 멤버가 아닌 이상 가변변수(여기서는 let으로 선언하는 변수)는 꼭 필요한 경우를 제외하고 지양하면 좋겠습니다. 물론 지금은 코드의 양 자체가 적어 문제 될 일이 없어보이긴 하지만 가변변수는 다른 곳에서 변경 될 수 있다는 여지를 항상 가지고 있습니다.
2. 명령형 제어문(여기서는 for문) 보다 선언적인 메서드 사용을 연습해보시면 좋겠습니다.


```suggestion
        const list = Array.from({ length: numLottos }, () => ths.generateRandomNumbers())
```